lib/kino/explorer.ex

```diff
@@ -161,11 +161,8 @@ defmodule Kino.Explorer do
   end
 
   defp build_summary(:categorical, column, series, has_groups, grouped, nulls) do
-    # TODO: Remove this when possible
-    # The main case that makes us need to use try/rescue here is when there are internal nils in a list
-    # For example: Series.from_list([[1, 2], [2, nil]]) will break on most_frequent and unique
-    try do
-      %{"counts" => top_freq, "values" => top} = most_frequent(series)
+    if most_frequent = most_frequent(series) do
```

Comment:
Instead of wrapping the whole block, could we instead exclude the "top" and "top freq" from summary but still include others?

Reply:
No, it also breaks on `unique`.